### ISSUE TITLE
`jsdoc/require-returns` does not recognize synonymous tags or preferences

### ISSUE THREAD

**Issue by scottmmjackson** [gajus/eslint-plugin-jsdoc#1129]
https://jsdoc.app/tags-returns.html

## Expected behavior

`@return` should either be acceptable or configurable as a preferred form of `@returns`. Some IDEs (notably IDEA) will autocomplete `@return`, and some Java teams will prefer this as well.

## Actual behavior

```
/**
 * My function.
 *
 * @return {string} Hello.
 */
function myFunction() { return "hello"; }
```

produces a `jsdoc/require-returns` error.

## ESLint Config

```json
{
  "extends": [
    "plugin:jsdoc/recommended"
  ],
}
```

## ESLint sample

See above

## Environment

- Node version: v20.3.1
- ESLint version: v5.16.0
- `eslint-plugin-jsdoc` version: ^18.11.0

**Comment by brettz9** [comment]
This is possible via settings:

```js
      settings: {
        jsdoc: {
          tagNamePreference: {
            returns: 'return',
          },
        },
      },
```

Closing as that should resolve, though feel free to comment further as needed.
